packages/orbit-components/src/hooks/useRandomId/__tests__/index.test.tsx

```diff
@@ -43,7 +42,7 @@ describe("useRandomId", () => {
       <OrbitProvider theme={{ ...theme }} useId={React.useId}>
         <Component />
       </OrbitProvider>,
-      { container, hydrate: true },
+      { container, hydrate: false },
```

Comment:
I couldn't use `renderToString` as this function is not supported anymore. Related to that, I had to set `hydrate` to `false` to keep test working and passing.